jd-geek-source/src/app/models/shipping.interface.ts

```diff
@@ -0,0 +1,6 @@
+export interface Shipping {
+    ground: string,
```

Comment:
Why are these lines intended four spaces but other files have lines intended with two spaces?

Reply:
Not entirely sure but I think I was fighting with the VS Code auto saving until I realized that the geek source project had it's on settings.  Some of the files I used the command to save with out formatting.  I think this file is going to go away anyway.